images.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <title> images</title>
+    <style>
+
+    h1
+    {
+      color: slateblue;
+      padding-left:800;
+      padding-top:20;
+      padding-bottom:20;
+      
+    }
+    
+    figure{
+      float:left;
+    }
+    
+    img{
+      width:250px;
+      height:250px;
+      padding: 5px; 
+      border-radius: 8px;
+     
+      
+    }
+    
+    figcaption{
+      text-decoration: darkslategray;
+      font-size: 30;
+      text-align:center;
+      
+    }
+       
+    
+    </style>
+      </head>
+    <body>
+        
+        <h1>Tourism</h1>
+      
+        <figure>
+            <img src="https://cdn.tourradar.com/s3/tour/original/79329_51048c99.jpg" >
+            <figcaption>Italy</figcaption>
+          </figure>
+        
+          <figure>
+            <img src="https://images.unsplash.com/photo-1508050919630-b135583b29ab?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=18cb84e149b5e364557dd9185282904d&w=1000&q=80"  >
+            <figcaption>Paris</figcaption>
+          </figure>
+         
+          <figure>
+            <img src="https://study-eu.s3.amazonaws.com/uploads/image/path/46/wide_fullhd_15317597976_b86a01bd5e_o.jpg"  >
+            <figcaption>Germany</figcaption>
+          </figure>
+          
+          <figure>
+            <img src="https://walkjapan.com/application/files/5514/5570/5986/hero-slide-japan.jpg"  >
+            <figcaption>Japan</figcaption>
+          </figure>
+        
+          <figure>
+            <img src="https://www.telegraph.co.uk/content/dam/Travel/Tours/India-xlarge.jpg"  >
+            <figcaption>India</figcaption>
+          </figure>
```

Comment:
Since this is a list of images with caption, you should have embedded these figure tags inside ul>li